Sources/RInAppMessaging/Models/CustomAttribute.swift

```diff
@@ -49,6 +49,13 @@ public class CustomAttribute: NSObject {
         self.type = .timeInMilliseconds
     }
 
+    @objc
```

Comment:
Is this new `init` function required with `public` scope since the ticket is to add only test cases?

Reply:
Oh ya, we don't need this function to be public, I changed this on this [commit](https://github.com/rakutentech/ios-inappmessaging/pull/253/commits/4c3a20be0732d3e646c28b7d0f44a12900a7b9d5).